### ISSUE TITLE
BackupPC 4.4.0 - Non-existing directory appearing in all backups

### ISSUE THREAD

**Issue by r0bb3d** [backuppc/backuppc#479]
Running BackupPC 4.4.0 packaged in Ubuntu 22.04 Jammy.

We run backups over rsync from other Ubuntu hosts and use the "BackupFilesOnly" directive to only backup specific directories. At one point I added a directory to that array and it was backed up for a few days like I wanted. Then I removed that directory from the source host and from the BackupFilesOnly array.

The issue is that the directory is gone from the source host and rotated out of all the backups by now. There is not a single reference to be found to this directory in BackupPC configs, however it keeps appearing in every new backup with a modification date from the date I removed it. When I click on it the directory is empty. When I check the "backup history" it is not listed in any backup number. So it's basically a ghost entry.

I'd say this is a bug.

But also any tips on how to resolve this myself are very welcome. Already restarted BackupPC daemon etc.

**Comment by r0bb3d** [comment]
Issue still persists

**Comment by r0bb3d** [comment]
Sadly I'm talking to myself here, but in case anybody stumbles upon this in a search. Our issue was a duplicate of [this issue](https://github.com/backuppc/backuppc/issues/301).

Basically I didn't add the "--delete" flag to the RsyncArgs because this wasn't the default or even needed in BackupPC v3 to not include deleted files in new backups. I kept our old config when I upgraded, but the behavior of the custom backuppc rsync binary seems to have changed in v4.
